src/VisualStudio/Core/Def/Implementation/TodoComments/VisualStudioTodoCommentsService.cs

```diff
@@ -22,10 +22,13 @@
 using Microsoft.CodeAnalysis.Remote;
 using Microsoft.CodeAnalysis.TodoComments;
 using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
+using Microsoft.VisualStudio.Utilities;
 using Roslyn.Utilities;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.TodoComments
 {
+    using Microsoft.CodeAnalysis.PooledObjects;
```

Comment:
curious why this is here?  it's imported outside the namespace already.  are there name conflicts with ```Microsoft.VisualStudio.Utilities```?

Reply:
that's the exact reason why unfortunately.

Reply:
Fixed!